storage-client/src/lib.rs

```diff
@@ -1,7 +1,7 @@
 pub mod client;
 
 use anyhow::Result;
-use arrow_array::RecordBatch;
+use arrow::record_batch::RecordBatch;
```

Comment:
I think what EE ppl agreed on is the arrow_array one? 
https://discord.com/channels/1198701067274297394/1200903142603239454/1201179689654419536

Reply:
🤔️ sounds reasonable but `istziio_client` already uses `arrow::record_batch::RecordBatch`. Shall we ask them to modify the interface?

Reply:
Could you talk to them about this first? We can discuss other things later tmr.